packages/core/src/common/command.ts

```diff
@@ -301,24 +355,21 @@ export class CommandRegistry implements CommandService {
     /**
      * Test whether there is an active handler for the given command.
      */
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    isEnabled(command: string, ...args: any[]): boolean {
+    isEnabled<Id extends string>(command: Id, ...args: Command.Arguments<Id>): boolean {
         return typeof this.getActiveHandler(command, ...args) !== 'undefined';
     }
 
     /**
      * Test whether there is a visible handler for the given command.
      */
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    isVisible(command: string, ...args: any[]): boolean {
+    isVisible<Id extends string>(command: Id, ...args: Command.Arguments<Id>): boolean {
         return typeof this.getVisibleHandler(command, ...args) !== 'undefined';
     }
 
     /**
      * Test whether there is a toggled handler for the given command.
      */
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    isToggled(command: string, ...args: any[]): boolean {
+    isToggled<Id extends string>(command: Id, ...args: Command.Arguments<Id>): boolean {
```

Comment:
I wouldn't mind seeing the returns here typed `unknown` and letting truthiness take care of itself (as is done for `Array.prototype.filter`), but requiring `boolean`s isn't wrong either.

Reply:
`unknown` would be more ergonomic and `boolean` is more explicit. But if we document checking for truthiness then `unknown` should do the trick indeed.